lib/MetaCPAN/Script/Role/External/Cygwin.pm

```diff
@@ -0,0 +1,72 @@
+package MetaCPAN::Script::Role::External::Cygwin;
+
+use v5.014;    # /r
+use Moose::Role;
+use namespace::autoclean;
+
+use List::Util      qw( shuffle );
+use Log::Contextual qw( :log );
+
+use MetaCPAN::Types qw( ArrayRef Int );
+
+has mirrors => (
+    is      => 'ro',
+    isa     => ArrayRef,
+    lazy    => 1,
+    builder => '_build_mirrors',
+);
+
+has mirror_timeout => (
+    is      => 'ro',
+    isa     => Int,
+    default => 10,
+);
+
+sub _build_mirrors {
+    my ($self) = @_;
```

Comment:
for consistency - please use the same style ( shift | @_) as in the next method (or change that one)